terraform_files/equinix-ci-machine/main.tf

```diff
@@ -27,7 +27,10 @@ resource "equinix_metal_device" "ci_devices" {
 
   provisioner "remote-exec" {
     inline = [
-      "cloud-init status --wait"
+      # Wait for cloud-init to complete.
+      # Ignore any errors because some equinix machines fail in cloud-init but
+      # it is without consequenses.
```

Comment:
wdym about "without consequences"? 
does it mean the code can continue running, only without the intermediate check that makes sure cloud-init finished? iiuc in this case we reach the timeout (or not? if the failure for those machines is immediate) for ``cloud-init status --wait``, ignore it, and continue with the rest of the instructions

Reply:
yeah, I mean it's not `cloud-init status --wait` can fail but we can continue nonetheless

Reply:
so it's fail after a timeout? (not sure if I fully understand)
either way, it's not critical

Reply:
no, equinix metal injects invalid user-data
https://redhat-internal.slack.com/archives/C0NFH1Y1X/p1676456507662149